src/components/Film/Film.css

```diff
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@300&display=swap');
+
+.single-film {
```

Comment:
You've said that you're not a design person, but I would say your work shows differently. You've added some great UX elements (the borders between the movie time, genres, and the date creation